DuckDuckGo/AutofillLoginDetailsView.swift

```diff
@@ -0,0 +1,290 @@
+                }
+            }
+        }.frame(height: 60)
+    }
+}
+
+struct ClearTextField: View {
+    @Binding var text: String
+    @State private var closeButtonVisible = false
+    
+    var body: some View {
+        HStack {
+            TextField("", text: $text) { editing in
+                closeButtonVisible = editing
+            } onCommit: {
+                closeButtonVisible = false
+            }
+            Spacer()
+            Image(systemName: "multiply.circle.fill")
+                .foregroundColor(.secondary)
+                .opacity(closeButtonOpacity)
+                .onTapGesture { self.text = "" }
+        }
+    }
+    
+    private var closeButtonOpacity: Double {
+        if text == "" || !closeButtonVisible {
+            return 0
+        }
+        return 1
+    }
+}
+
+private struct CopyablePasswordCell: View {
+    @State private var id = UUID()
+    let title: String
+    let password: String
+    @Binding var selectedCell: UUID?
+    @Binding var isPasswordHidden: Bool
+    let action: () -> Void
+    
+    var body: some View {
+        ZStack {
+            HStack {
+                VStack(alignment: .leading, spacing: 6) {
+                    Text(title)
+                        .label3AltStyle()
+                        .disabled(true)
+                    
+                    HStack {
+                        Text(password)
+                            .label3Style(design: .monospaced)
+                    }
+                }
+                Spacer()
+            }
+            .copyable(isSelected: selectedCell == id, menuTitle: title, menuAction: {
+                self.action()
+            }, tapAction: {
+                self.selectedCell = self.id
+            })
+            
+            HStack {
+                Spacer()
+                Button {
+                    isPasswordHidden.toggle()
+                    self.selectedCell = nil
+                } label: {
+                    HStack {
+                        Spacer()
+                        Image(isPasswordHidden ? "ShowPasswordEye": "HidePasswordEye")
+                            .foregroundColor(.primary)
+                    }
+                    .contentShape(Rectangle())
+                    .frame(width: 50, height: 50)
+                }
+                .buttonStyle(.plain)
+                .background(BackgroundColor(isSelected: selectedCell == id).color)
+                // Prevent taps from being forwarded
+            }
+        }
+        .selectableBackground(isSelected: selectedCell == id)
+    }
+
+}
+
+private struct CopyableCell: View {
+    @State private var id = UUID()
+    let title: String
+    let subtitle: String
+    @Binding var selectedCell: UUID?
+    let action: () -> Void
+        
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading, spacing: 6) {
+                Text(title)
+                    .label3AltStyle()
+                    .disabled(true)
```

Comment:
It was required because without it, the tap recognizer for the MenuItem wasn't being triggered, but this was later fixed some other way and the disable is a left over not needed anymore.
Removed, thanks